src/tippy.scss

```diff
@@ -4,6 +4,7 @@
 
 .tippy-box {
     @apply bg-transparent;
+    max-width: none !important;
```

Comment:
https://github.com/atomiks/tippyjs/issues/451

Reply:
How tf did you stumble on that Issue?